regression/cbmc/enum_is_in_range/enum_test10.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+enum_test10.c
+
+^EXIT=6$
+^SIGNAL=0$
+^file enum_test10.c line \d+ function main: __CPROVER_enum_is_in_range expects enum, but \(i\) has type `signed int`$
```

Comment:
This test ought to live in the `regression/ansi-c/` folder, I'd say. This will require modifying `EXIT=6` to `EXIT=(1|64)`